cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/CompanyDTO.java

```diff
@@ -206,11 +214,29 @@ public void setNotificationSecretToken( String notificationSecretToken )
         this.notificationSecretToken = notificationSecretToken;
     }
 
+    public List<UrlEntityDTO> getUsers()
+    {
+        return users;
+    }
+
+    public void setUsers( List<UrlEntityDTO> users )
+    {
+        this.users = users;
+    }
+
+    /**
+     * @deprecated by {@link #getUsers()}
+     */
+    @Deprecated
     public List<UrlEntityDTO> getVendors()
     {
         return vendors;
     }
 
+    /**
+     * @deprecated by {@link #setUsers(List)}
+     */
+    @Deprecated
     public void setVendors( List<UrlEntityDTO> vendors )
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)